img/foxsi_image_det.pro

```diff
@@ -34,14 +36,15 @@
 ;-
 
 FUNCTION FOXSI_IMAGE_DET, DATA, ERANGE = ERANGE, TRANGE = TRANGE, $
-                          THR_N = THR_N, KEEPALL = KEEPALL, $
+                          THR_N = THR_N, KEEPALL = KEEPALL, cdte=cdte, $
                           YEAR=YEAR, flatfield=flatfield, STOP = STOP
     COMMON FOXSI_PARAM
-	default, erange, [4.,15.]
+	  default, erange, [4.,15.]
```

Comment:
Is this "space" before the word default actually needed?

Reply:
No, I will remove it.